foundation/foundation-upgrade/src/test/resources/test-files/v1_8_0/ExtensionsSecurityProjectsMigration/validation/IngestValidation.java

```diff
@@ -0,0 +1,81 @@
+package org.boozallen.aissemble;
+
+/*-
+ * #%L
+ * authzforce1::Pipelines::Spark Pipeline
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+
+import javax.enterprise.context.ApplicationScoped;
+
+import com.boozallen.aissemble.security.authorization.AissembleKeyStore;
+import com.boozallen.aissemble.security.authorization.AissembleKeycloakSecureTokenServiceClient;
+import com.boozallen.aissemble.security.authorization.AissembleSimpleSecureTokenServiceClient;
+import com.boozallen.aissemble.security.authorization.policy.AissembleAttribute;
+import com.boozallen.aissemble.security.authorization.policy.AissembleAttributeProvider;
+import com.boozallen.aissemble.security.exception.AissembleSecurityException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Instant;
+import java.util.Map;
+
+
+/**
+ * Performs the business logic for Ingest.
+ *
+ * Because this class is {@link ApplicationScoped}, exactly one managed singleton instance will exist
+ * in any deployment.
+ *
+ * GENERATED STUB CODE - PLEASE ***DO*** MODIFY
+ *
+ * Originally generated from: templates/data-delivery-spark/synchronous.processor.impl.java.vm
+ */
+@ApplicationScoped
+public class Ingest extends IngestBase {
+
```

Comment:
It's not a big deal since this is a test file, but the name of the Class should match the name of the Java file.

Reply:
The reason why they don't match is because as part of the assetTrue(), we compare the changes that were made to the input file against the validated file. The names are just difference just to differentiate the functions of the files.